pkg/kube/client.go

```diff
@@ -629,3 +646,90 @@ func (c *Client) watchPodUntilComplete(timeout time.Duration, info *resource.Inf
 
 	return err
 }
+
+func (c *Client) getRelationPods(infos []*resource.Info) map[string][]api.Pod {
+	var objPods = make(map[string][]api.Pod)
+
+	for _, value := range infos {
+		objPods, _ = c.getSelectRelationPod(value, objPods)
+	}
+	return objPods
+}
+
+func (c *Client) getSelectRelationPod(info *resource.Info, objPods map[string][]api.Pod) (map[string][]api.Pod, error) {
+	if info == nil {
+		return objPods, nil
+	}
+
+	err := info.Get()
+	if err != nil {
+		return objPods, err
+	}
+
+	versioned, err := c.AsVersionedObject(info.Object)
+	if runtime.IsNotRegisteredError(err) {
+		return objPods, nil
+	}
+	if err != nil {
+		return objPods, err
+	}
+
+	selector, err := getSelectorFromObject(versioned)
+
+	log.Printf("getSelectRelationPod selector: %+v", selector)
```

Comment:
With these log statements, can you make sure to use `c.Log` (you can see examples of this throughout the other functions)